package.json

```diff
@@ -33,7 +33,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "./scripts/pre-commit.sh"
```

Comment:
What's the reason behind this?

Reply:
Originally I was performing some configuration actions on `pre-commit` which required a dedicated file, but eventually these were moved to `postinstall.sh`. I thought there might be a chance it would be useful to hang onto a dedicated file, but I'll revert this too!